src/compile/miniconstants.rs

```diff
@@ -4,284 +4,85 @@
 
 //!Creates a fixed list of globally accessible constants.
 
+use crate::compile::CompileError;
 use crate::evm::{builtin_contract_path, AbiForContract};
 use crate::uint256::Uint256;
+use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
+use std::fs::File;
+use std::io::Read;
+use std::path::Path;
+
+#[derive(Clone, Default, Debug, Serialize, Deserialize, Eq, PartialEq)]
+pub struct ConstantsFile {
+    integer: Vec<(String, u64)>,
+    hex: Vec<(String, String)>,
+    contract: Vec<String>,
+}
 
 pub static ARBOS_VERSION: u64 = 5;
```

Comment:
We'll need to get the version from the file if we need it. This constant should be removed since since it won't be synchronized with the json file